lms/djangoapps/teams/static/teams/js/views/topic_card.js

```diff
@@ -46,7 +46,9 @@
                         gettext('View Teams in the %(topic_name)s Topic'),
                         { topic_name: this.model.get('name') }, true
                     ));
-                    return '<span class="sr">' + screenReaderText + '</span><i class="icon fa fa-arrow-right" aria-hidden="true"></i>';
+                    return '<span class="sr">' +
```

Comment:
There were 2 or 3 places that I had to concat the string in separate lines instead of in the same line because the quality tests were giving me an error with the lines being too long. I'm not sure how those lines got through those tests before